views/js/qtiCreator/test/MathEditor/test.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>QtiElements Test - QTI item model information model definition </title>
+        <link rel="stylesheet" type="text/css" href="/tao/views/js/lib/qunit/qunit.css">
+        <script type="text/javascript" src="/tao/views/js/lib/require.js"></script>
+        <script type="text/javascript" src="/tao/views/js/lib/qunit/qunit.js"></script>
+        <script type="text/javascript" src="/tao/views/js/lib/blanket/blanket.min.js" data-cover-only="/taoQtiItem/views/js/qtiCreator/editor/MathEditor.js"></script>
+        <script  type="text/javascript">
+            QUnit.config.autostart = false;
+            require(['/tao/ClientConfig/config'], function(){
+                require(['taoQtiItem/qtiCreator/test/MathEditor/test'], function(){
+                    QUnit.start();
+                });
+            });
+        </script>
+        <!-- http://xxx/taoQtiItem/views/js/qtiCreator/test/MathEditor/test.html -->
+    </head>
+    <body>
+        <div id="qunit"></div>
+        <div id="qunit-fixture"></div>
```

Comment:
The fixture element is reset at the end of each test. `QUnit.start` should be called then be called at the end (after all asserts).